formfyxer/lit_explorer.py

```diff
@@ -397,52 +409,22 @@ def normalize_name(
     this_field: str,
     tools_token: Optional[str] = None,
 ) -> Tuple[str, float]:
-    """Add hard coded conversions maybe by calling a function
-    if returns 0 then fail over to ML or other way around poor prob -> check hard-coded.
-    Retuns the new name and a confidence value between 0 and 1"""
+    """
+    Normalize a field name, if possible to the Assembly Line conventions, and if
+    not, to a snake_case variable name of appropriate length.
+
+    HACK: temporarily all we do is re-case it and normalize it using regex rules.
+    Will be replaced with call to LLM soon.        
+    """
+    
     if this_field not in included_fields:
         this_field = re_case(this_field)
-        out_put = regex_norm_field(this_field)
-        conf = 1.0
-        if out_put == this_field:
-            params = []
-            for item in jurisdictions:
-                if jur == item:
-                    params.append(1)
-                else:
-                    params.append(0)
-            for item in groups:
-                if group == item:
-                    params.append(1)
-                else:
-                    params.append(0)
-            params.append(n)
-            params.append(per)
-            for vec in vectorize(this_field, tools_token=tools_token):
-                params.append(vec)
-            for item in included_fields:
-                if last_field == item:
-                    params.append(1)
-                else:
-                    params.append(0)
-            pred = clf_field_names.predict([params])
-            prob = clf_field_names.predict_proba([params])
-            conf = prob[0].tolist()[prob[0].tolist().index(max(prob[0].tolist()))]
-            out_put = pred[0]
-    else:
-        out_put = this_field
-        conf = 1
-    if out_put in included_fields:
-        if conf >= 0:
-            return (
-                "*" + out_put,
-                conf,
-            )  # this * is a hack to show when something is in the list of known fields later. I need to fix this
-        else:
-            return reformat_field(this_field, tools_token=tools_token), conf
-    else:
-        return reformat_field(this_field, tools_token=tools_token), conf
+        this_field = regex_norm_field(this_field)
 
+    if this_field in included_fields:
+        return f"*{this_field}", 1.0
```

Comment:
Minor nit; we aren't very confident in this at all actually (lol), so it should be lower. I don't think is actually used anywhere, so changing it shouldn't hurt.

```suggestion
        return f"*{this_field}", 0.01
```